bazel/repository_locations.bzl

```diff
@@ -303,6 +303,33 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         release_date = "2020-08-25",
         cpe = "N/A",
     ),
+    skywalking_data_collect_protocol = dict(
```

Comment:
Can we remove this dep?

Reply:
cpp2sky depends on data collect protocol so we can't remove this.

Reply:
Do any of these APIs appear get used by protos in `api/envoy/`?

Reply:
No. `skywalking_data_collect_protocol` is completely independent of envoy. Only this tracer dependency depends on these APIs.